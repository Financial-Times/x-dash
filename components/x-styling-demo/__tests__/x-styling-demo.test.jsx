const snapshots = require('@financial-times/x-test-utils/snapshots');
const stories = require('../stories');

describe('x-styling-demo', () => {
	snapshots(stories);
});
