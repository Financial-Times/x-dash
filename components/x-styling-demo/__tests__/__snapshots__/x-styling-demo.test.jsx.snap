// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`x-styling-demo renders a default Styling 1`] = `
<button
  className="Button_button__vS7Mv"
>
  Click me!
</button>
`;
